cfme/cloud/keypairs.py

```diff
@@ -184,6 +185,41 @@ def download_private_key(self):
         view.toolbar.configuration.item_select('Download private key')
         view.flash.assert_no_error()
 
+    def set_ownership(self, owner=None, group=None, click_cancel=False, click_reset=False):
```

Comment:
_Optional_
```suggestion
    def set_ownership(self, owner=None, group=None, cancel=False, reset=False):
```

Reply:
**Required**

Entity/collection methods need to be consistent with their kwargs.

Reply:
I see that common/vm has these kwargs - but its inconsistent with the rest of the framework kwargs for these methods.